pkgs/os-specific/linux/firmware/fwupd/fix-paths.patch

```diff
@@ -4,5 +4,5 @@ index c7a430c0..e69de29b 100644
 +++ b/data/builder/meson.build
 @@ -1,3 +0,0 @@
 -install_data('README.md',
--  install_dir : join_paths(localstatedir, 'lib', 'fwupd', 'builder')
+-  install_dir : join_paths(datadir, 'doc', 'fwupd', 'builder')
 -)
```

Comment:
Is this patch necessary? Writing to datadir should not cause issues.

Reply:
Ack, wasn't thinking when I opened this. I agree it doesn't seem necessary, will fix it up.